lms/views/api/exceptions.py

```diff
@@ -157,6 +161,7 @@ def external_request_error(self):
 
     @exception_view_config(context=OAuth2TokenError)
     def oauth2_token_error(self):
+        LOG.debug("Exception view for: OAuth2TokenError")
```

Comment:
I'm not sure that I'd bother with this, to be honest

Reply:
I reckon if we logged every request in the local server (and probably the status code) this won't be needed but without it now the server logs would be silent for this case which can be confusing.